packml_ros/src/packml_ros.cpp

```diff
@@ -308,6 +339,24 @@ void PackmlRos::publishStats()
   stats_pub_.publish(stats);
 }
 
+void PackmlRos::publishStatsTransactionCb(const ros::TimerEvent &timer_event)
+{
+  // Check if stats_transaction_publish_period changed
+  float stats_transaction_publish_period_new;
```

Comment:
`float` type here, but `double` in the class reference can lead to loss of precision warnings.

Reply:
Thank for catching that, everything is double now.